src/NzbDrone.Core/Http/MTeamTpHttpRequestInterceptor.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using NzbDrone.Common.Http;
+
+namespace NzbDrone.Core.Http
+{
+    public class MTeamTpHttpRequestInterceptor : IHttpRequestInterceptor
+    {
+        private string Host => "kp.m-team.cc";
+
+        private string UserAgent => "Mozilla/5.0 (X11; Linux x86_64; rv:94.0) Gecko/20100101 Firefox/94.0";
```

Comment:
Useragent can already be set via the definition

Reply:
fix m team only file https://github.com/Prowlarr/Prowlarr/pull/1103/files/f659bf7bb118d8892608f8c8d65ed975ba6b345f#diff-fc204774368bcfdaddf157f615f145fec411d3f65f2e83f61b4e5748cfca20cb